browser/js/create-game/create-game.controller.js

```diff
@@ -41,5 +42,16 @@ app.controller('CreateGameCtrl', function($scope, $mdDialog){
             scope: $scope
         });
     }
+    $scope.getMatches = function() {
+        UserFactory.autocomplete($scope.searchText)
+        .then(users=>$scope.foundMatches = users)
+        .catch(err=>$log.error)
+
+    }
+
+    $scope.create = function(){
+        GameFactory.createGame($scope.comm)
+        .then(gameId=>$state.go('u.game', {gameId:gameId}))
```

Comment:
Nice es6 trick when a key and value happen to be the same: `{gameId: gameId}` can also be expressed as `{gameId}`. See [here](http://www.benmvp.com/learning-es6-enhanced-object-literals/#property-value-shorthand) for more.